### PR TITLE
refactor/role: changed the edit and delete permission

### PR DESCRIPTION
**Pull request by JessBloem**
## Pull Request

**Description:**

Update the permissions 

**Type of Change:**

- [ ] Bug Fix
- [ ] New Feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Documentation Update

**Checklist:**

- [x] My code follows the coding standards of this project.
- [x] I have performed a self-review of my code.
- [x] I have added/updated relevant documentation.
- [x] My changes generate no new warnings or errors.
- [ ] I have tested my changes.

**Screenshots (if applicable):**

[Add screenshots to help reviewers understand the changes visually.]

**Reviewers:**

- @Hamza-Mokiwa 
- @CodeHermez 
- @KumbiraiShonhiwa 

**Additional Notes:**

[Any additional information that may be helpful for reviewers.]

### CODE REVIEW

**Comment by JessBloem** [comment]
Please don't merge it, I will merge it after it was approved

**Review by CodeHermez — APPROVED** [review]
👍🏿

**Review by Hamza-Mokiwa — APPROVED** [review]
Yes.

**Review by KumbiraiShonhiwa — APPROVED** [review]

**Pull request merged**
